src/System.Private.ServiceModel/tools/test/HelixLabRun/launchWCFOpenBuilds.cmd

```diff
@@ -0,0 +1,88 @@
+REM
+REM This script sync to the latest WCF source, build and submit Helix lab runs.
+REM Arguments:
+REM %1: Lab scripts location
+REM %2: WCF local git source location
+REM %3: Response files location
+
+@echo off
+setlocal
+
+set BridgeHost=
+pushd %2
+pushd bin\wcf\tools\Bridge
+if [%ERRORLEVEL%]==[0] (
+    call Bridge.exe -stop
+    popd
+)
+
+#REM remove bin directory to ensure a rebuild
+call git clean -xdf
+if NOT [%ERRORLEVEL%]==[0] (
+    echo ERROR: An error occurred while cleanning Git.
+    exit /b
+  )
+
+REM sync
+git checkout master
+if NOT [%ERRORLEVEL%]==[0] (
+    echo ERROR: An error occurred while checking out master.
+    exit /b
+  )
+git pull
```

Comment:
You should never be doing git pull operations as this does a git fetch and then a git merge. You will put the repo into a weird state (i.e., a state where they will no longer track origin/master and upstream/master) by doing this. 

It is _reasonably_ safe if you choose to do this on a lab box, but still not recommended. 
You're better off doing a git fetch then git merge --ff-only to ensure that there's no merge commits in your repo.